### PR TITLE
feat(apple): Point out enableTracing in Getting Started 

### PR DESCRIPTION
**Pull request by brustolin**
## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes

Added `enableTracing` in the "Getting Started" section for cocoa so the user could see more transactions out of the box.

More details at #5730

### CODE REVIEW

File: src/platform-includes/getting-started-config/apple.mdx [hunk 1]
```diff
@@ -25,6 +25,10 @@ func application(_ application: UIApplication,
     [SentrySDK startWithConfigureOptions:^(SentryOptions *options) {
         options.dsn = @"___PUBLIC_DSN___";
         options.debug = YES; // Enabled debug when first installing is always helpful
+
+        // Enable tracing to capture 100% of transactions for performance monitoring.
+        // If you want to choose the sampling rate, you can use 'options.tracesSampleRate'
```
Comment (shanamatthews) [inline src/platform-includes/getting-started-config/apple.mdx:30]: In other SDKs we usually mention something like "We recommend adjusting this value in production" for `tracesSampleRate` to help prevent users from running out of quota.

Maybe we could say something like:

```suggestion
        // Enable tracing to capture 100% of transactions for performance monitoring.
        // Use 'options.tracesSampleRate' to set the sampling rate. 
        // We recommend setting a sample rate in production.
```

File: src/platform-includes/getting-started-config/apple.mdx [hunk 2]
```diff
@@ -42,6 +46,10 @@ struct SwiftUIApp: App {
         SentrySDK.start { options in
             options.dsn = "___PUBLIC_DSN___"
             options.debug = true // Enabled debug when first installing is always helpful
+
+            // Enable tracing to capture 100% of transactions for performance monitoring.
+            // If you want to choose the sampling rate, you can use 'options.tracesSampleRate'
```
Comment (shanamatthews) [inline src/platform-includes/getting-started-config/apple.mdx:51]: // Enable tracing to capture 100% of transactions for performance monitoring.
            // Use 'options.tracesSampleRate' to set the sampling rate. 
            // We recommend setting a sample rate in production.
```

**Review by shanamatthews — APPROVED** [review]

**Comment by shanamatthews** [comment]
Hey @brustolin - is this ready for merge?

**Comment by brustolin** [comment]
Oh, yes. Forgot about this.

**Pull request merged**